interactions_test.go

```diff
@@ -391,3 +391,76 @@ func TestInteractionCallback_InteractionTypeBlockActions_Unmarshal(t *testing.T)
 		cb.BlockActionState.Values["section_block_id"]["multi_convos"].SelectedConversations,
 		[]string{"G12345"})
 }
+
+func TestInteractionCallback_ValidateToken(t *testing.T) {
```

Comment:
`(InteractionCallback).ValidateToken` returns true if both `verificationTokens` and `(InteractionCallback).Token` are an empty.
Is this the intended behavior?